server/events_controller_e2e_test.go

```diff
@@ -328,6 +328,7 @@ func TestGitHubWorkflow(t *testing.T) {
 			w := httptest.NewRecorder()
 			When(githubGetter.GetPullRequest(AnyRepo(), AnyInt())).ThenReturn(GitHubPullRequestParsed(headSHA), nil)
 			When(vcsClient.GetModifiedFiles(AnyRepo(), matchers.AnyModelsPullRequest())).ThenReturn(c.ModifiedFiles, nil)
+			// *** need azuredevops tests here ***
```

Comment:
@mcdafydd these tests right now are only for GitHub so AZDO won't be the only vcs without tests. The purpose of these was mostly to test internal atlantis behaviour so I think it's okay if you don't have AZDO.

Reply:
Great, thanks!!